contracts/carbon/Strategies.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.19;
 import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import { EnumerableSetUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/structs/EnumerableSetUpgradeable.sol";
 import { MathUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/math/MathUpgradeable.sol";
+import { SafeMathUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/math/SafeMathUpgradeable.sol";
```

Comment:
Not important, but since these libraries aren't really "upgradeable" (they are equivalent to the non-upgradeable versions), you may consider importing them as `SafeMath` in the future (i.e., `import { SafeMathUpgradeable as SafeMath } ...`